contracts/core/EntryPoint.sol

```diff
@@ -763,4 +763,10 @@ contract EntryPoint is IEntryPoint, StakeManager, NonceManager, ReentrancyGuard,
             data := offset
         }
     }
+
+    /// @inheritdoc IEntryPoint
+    function delegateAndRevert(bytes calldata data) external {
+        (bool success, bytes memory ret) = address(this).delegatecall(data);
```

Comment:
What is the difference between `call` and `delegatecall` in this case? I don't think the EntryPoint ever relies on the `msg.sender`, does it?